src/adapter/breakpoints.ts

```diff
@@ -46,16 +48,7 @@ export class Breakpoint {
   async _notifyResolved(): Promise<void> {
     if (!this._resolvedUiLocation)
       return;
-    this._manager._dap.breakpoint({
-      reason: 'changed',
-      breakpoint: {
-        id: this._dapId,
-        verified: true,
-        source: await this._resolvedUiLocation.source.toDap(),
-        line: this._resolvedUiLocation.lineNumber,
-        column: this._resolvedUiLocation.columnNumber
-      }
-    });
+    await this._manager.notifyBreakpointResolved(this._dapId, await this._resolvedUiLocation);
```

Comment:
```suggestion
    await this._manager.notifyBreakpointResolved(this._dapId, this._resolvedUiLocation);
```

await is extraneous here

Reply:
Fixed